app.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask, request, jsonify
+import mysql.connector
+
+app = Flask(__name__)
+
+#MySQL Connection Configuration
+users_db = mysql.connector.connect(
+    host="mysql",
```

Comment:
it's better to make this list of values into enviroment variables, and pass them to the 2 containers